charts/substra-backend/values.yaml

```diff
@@ -696,6 +694,8 @@ oidc:
     loginValidityDuration: 3600
     ## @param oidc.users.channel The channel to assign OIDC users to (mandatory)
     channel: null
+    ## @param oidc.users.requireApproval Is not compatible with default channel
```

Comment:
Make more explicit what this parameter is doing, in addition to tell with which option it is imcompatible

Reply:
no they should incompatible, the default channel is assigned to every OIDC created user. While the requireApproval should allow the OIDC to create users with no channel